src/utils/estimertRefusjon.ts

```diff
@@ -3,9 +3,11 @@ const gbeløp: number = 101351; // g-beløp per 29. januar 2021
 const estimertRefusjon = (månedsinntekt: number, ukedager: number): number => {
   const årslønn: number = Math.min(månedsinntekt * 12, gbeløp * 6);
 
-  const dagsats: number = årslønn / 260;
+  const dagsats: number = Math.round(årslønn / 260);
 
-  return Math.round(dagsats * ukedager * 0.7);
+  const vektetDagsats: number = Math.round(dagsats * 0.7);
+
+  return vektetDagsats * ukedager;
```

Comment:
Det er ikke bare sluttsummen som skal rundes av? Skulle den ikke rundes av til to desimaler?
Information overload.....

Reply:
Stemmer. Dagsatsen skal rundes av til nærmeste halve/hele krone. Her rundes av til hele krone alltid.

Reply:
3) Dagsats = Årslønn / 260. Dagsats avrundes til helt kronebeløp. Feks avrundes 1981,49,- til 1981,- mens 1981,50,- avrundes til 1982,-. 
4) Den avrundede dagsatsen ganges med 0,7 og den dagsatsen man da får avrundes igjen til et helt kronebeløp

Reply:
Skal avrundes til hele kroner